client/src/featureform/serving.py

```diff
@@ -712,6 +857,26 @@ def parse_proto_value(value):
     return getattr(value, value.WhichOneof("value"))
 
 
+def proto_type_to_np_type(value):
+    type_mapping = {
+        "str_value": str,
+        "int_value": np.int32,
+        "int32_value": np.int32,
+        "int64_value": np.int64,
+        "float_value": np.float32,
+        "double_value": np.float64,
+        "bool_value": bool,
+    }
+    return type_mapping[value.WhichOneof("value")]
+
+
+def get_numpy_array_type(types):
```

Comment:
i'd throw a parameterized unit test at this function. reads like it could easily be a source of a hidden logic bug.